static/stats/js/stats.js

```diff
@@ -216,7 +216,12 @@ $(document).ready(function() {
                 sectionChart.data = result;
                 document.getElementById("select-section-displayByCountry").style.display = "block";
                 var ctx = document.getElementById("answers-per-section");
-                sectionChart.canvas = radarChart(labels,data_sets,ctx);
+                document.getElementById('select-section-graphType').style.display = "block";
+                if (document.getElementById('section_graphType').value == "radar") {
+                    sectionChart.canvas = radarChart(labels,data_sets,ctx);
+                } else {
+                    sectionChart.canvas = bubbleChart(labels,data_sets,ctx);
```

Comment:
`Is the current data_set response suitable for the charts rendering ?`

No, I mentioned in MM the need to add a new variable via the (Python file)[https://github.com/NC3-LU/Fit4Cybersecurity/blob/surveySMEDemand/stats/views.py] to add the sector between country and size in the (query)[https://github.com/NC3-LU/Fit4Cybersecurity/blob/surveySMEDemand/stats/views.py].

The idea would be to change the dataset according to the chart type, but I need to access the new variable first and I don't know how to do that.

I thought it would be a good idea to work on this PR until it works with both modified files and succeeds. 

Let me know here or IRL if you see a better way to proceed.

`CodeStyle note of space after comma.`

Thanks, I am going to change it.